CMakeLists.txt

```diff
@@ -141,6 +153,9 @@ if(MSVC)
     # https://gitlab.kitware.com/cmake/cmake/-/issues/18837
     add_compile_options(/Zc:__cplusplus)
 
+    # https://devblogs.microsoft.com/cppblog/announcing-full-support-for-a-c-c-conformant-preprocessor-in-msvc/
+    add_compile_options(/Zc:preprocessor)
```

Comment:
Were there actual changes that needed this?

Reply:
This is also going to be added in another PR (bgfx bump) to separate changes

Reply:
https://github.com/BabylonJS/BabylonNative/pull/1408